source/renderer/app/stores/SidebarStore.js

```diff
@@ -65,7 +65,7 @@ export default class SidebarStore extends Store {
   @action _configureCategories = () => {
     const {
       isFlight,
-      environment: { isDev, isMainnet },
+      // environment: { isDev, isMainnet },
```

Comment:
Is this no longer needed?

Reply:
This is only because for testing purposes you need to enable it with true (see line 84 of that file) and isDev, isMainnet gets obsolete and the lint throws an error. That is why @lucas-barros commented that line. I guess :)

Reply:
Thats right, I had to comment to enable voting on testnet.